account.go

```diff
@@ -61,37 +61,54 @@ const (
 
 // AccountParams are the parameters allowed during account creation/updates.
 type AccountParams struct {
-	Params
-	Country, Email, DefaultCurrency, Statement, BusinessName, BusinessUrl,
-	BusinessPrimaryColor, SupportPhone, SupportEmail, SupportUrl,
-	FromRecipient, PayoutStatement string
-	ExternalAccount                      *AccountExternalAccountParams
-	LegalEntity                          *LegalEntity
-	PayoutSchedule                       *PayoutScheduleParams
-	DebitNegativeBal, NoDebitNegativeBal bool
-	TOSAcceptance                        *TOSAcceptanceParams
-	Type                                 AccountType
+	Params               `form:"*"`
+	Country              string                        `form:"country"`
+	Email                string                        `form:"email"`
+	DefaultCurrency      string                        `form:"default_currency"`
+	Statement            string                        `form:"statement_descriptor"`
+	BusinessName         string                        `form:"business_name"`
```

Comment:
Minor but based on the size of this PR already, should we bite the bullet and do alphabetical ordering? I am happy to do this one if that helps you. Might be the right time to also use longer names once and for all.

Reply:
Totally agree, but mind if I do this on a separate PR?

And yeah, I gave serious thought to just expanding all the names. The only reason I didn't do it was to try and minimize what's here a little bit. I think we can do this later as part of a minor version bump because I'd leave the old field names in place for the time being (no sense pulling out the floor from under people for a cosmetic change like that).